docs/references/uvh5_memo.tex

```diff
@@ -140,51 +152,71 @@ \subsection{Required Parameters}
 \item \textbf{Nblts}: \textit{int} The number of baseline-times (i.e., the
   number of spectra) present in the data. Note that this value need not be equal
   to Nbls $\times$ Ntimes. (\textit{Nblts})
-\item \textbf{Nfreqs}: \textit{int} The number of frequency channels in the
-  data. (\textit{Nfreqs})
+\item \textbf{Nspws}: \textit{int} The number of spectral windows present in the
+  data. (\textit{Nspws})
+\item \textbf{Nfreqs}: \textit{int} The total number of frequency channels in
+  the data across all spectral windows. (\textit{Nfreqs})
 \item \textbf{Npols}: \textit{int} The number of polarization products in the
   data. (\textit{Npols})
 \item \textbf{Ntimes}: \textit{int} The number of time samples present in the
   data. (\textit{Ntimes})
-\item \textbf{Nspws}: \textit{int} The number of spectral windows present in the
-  data. (\textit{Nspws})
 \item \textbf{uvw\_array}: \textit{float} An array of the uvw-coordinates
   corresponding to each observation in the data. This is a two-dimensional array of
-  size (Nblts, 3). (\textit{uvw\_array})
+  size (Nblts, 3). Units are in meters. (\textit{uvw\_array})
 \item \textbf{time\_array}: \textit{float} An array of the Julian Date
   corresponding to the center of an integration. This is a one-dimensional array
   of size Nblts. (\textit{time\_array})
 \item \textbf{integration\_time}: \textit{float} An array of the length of time
   in seconds of an integration. This is a one-dimensional array of size
   Nblts. (\textit{time\_array})
 \item \textbf{freq\_array}: \textit{float} An array of the frequencies stored in
-  the file in Hertz. This is a two-dimensional array of size (Nspws,
-  Nfreqs). (\textit{freq\_array})
+  the file in Hertz. This is a one-dimensional array of size
+  (Nfreqs). (\textit{freq\_array})
```

Comment:
Maybe we should list the changes (including this one) in the version history section so that if someone needs to read an older version they know what to expect?